Sources/Optimizely/OptimizelyClient.swift

```diff
@@ -1005,6 +987,13 @@ extension OptimizelyClient {
                                           completionHandler: completionHandler)
     }
     
+    @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
+    func fetchQualifiedSegments(userId: String,
+                                options: [OptimizelySegmentOption]) async throws -> [String]? {
+        return try await odpManager.fetchQualifiedSegments(userId: userId,
+                                                           options: options)
+    }
+    
```

Comment:
no need async version. it's not public api.

Reply:
Done